src/main/java/com/google/sps/servlets/ReasonServlet.java

```diff
@@ -1,27 +1,65 @@
 package com.google.sps.servlets;
 
+import static com.google.cloud.spanner.TransactionRunner.TransactionCallable;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.ReadOnlyTransaction;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.TransactionContext;
+import com.google.gson.Gson;
 import java.io.IOException;
 import java.time.Instant;
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that sends reason for use to /reason. (will change to write to database)*/
+/** Servlet that writes Audit to database*/
 @WebServlet("/reason")
 public final class ReasonServlet extends HttpServlet {
+
+  DatabaseClient dbClient;
+  String queryString;
+  String selectedDatabase;
```

Comment:
No need for this class variable since we are always using "example-db", you can just delete it and when initiate dbClient, just do

this.dbClient = DatabaseConnector.getInstance().getDbClient("example-db");

Reply:
Done